freedesktop/com.github.blackhole89.notekit.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.github.blackhole89.notekit.desktop</id>
```

Comment:
```suggestion
	<id>com.github.blackhole89.notekit</id>
```

Reply:
You also need a `<launchable type="desktop-id">com.github.blackhole89.notekit.desktop</launchable>`

Reply:
Is that in spite of changing the `<id>` to `com.github.blackhole89.notekit`, or should that also be `<launchable type="desktop-id">com.github.blackhole89.notekit</launchable>`?

Reply:
(This github code review thing is making my head spin. Is the idea that I click "commit suggestion" on the appropriate subthreads *before* merging the whole PR?)

Reply:
launchable tag tells the software center how to run your app once installed, in your case it would be a desktop file name which should `$app-id.desktop`